ux.symfony.com/templates/recipe.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html.twig' %}
+
+{% set meta = {
+    title: 'Cookbook',
+    title_suffix: ' - Symfony UX',
+    description: 'Symfony UX cookbook - Concrete exemple to understant all the concept and components of Symfony UX',
```

Comment:
typo: => Concrete exeample to understantd all the concepts and components of Symfony UX